### PR TITLE
skins tab removed

### PR DESCRIPTION
**Pull request by Shriram-Patil-Arista**
before : 
Screenshot from 2024-02-13 16-42-02

after:
Screenshot from 2024-02-13 16-39-37

### CODE REVIEW

File: uvm/servlets/admin/config/administration/Main.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ Ext.define('Ung.config.administration.Main', {
         { xtype: 'config-administration-admin' },
         { xtype: 'config-administration-certificates' },
         { xtype: 'config-administration-snmp' },
-        { xtype: 'config-administration-skins' },
+        //{ xtype: 'config-administration-skins' },
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/administration/Main.js:25]: Let's just delete it, not comment it out.

File: uvm/servlets/admin/config/administration/MainController.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ Ext.define('Ung.config.administration.MainController', {
         Ext.Deferred.sequence([
             Rpc.asyncPromise('rpc.adminManager.getSettings'),
             Rpc.asyncPromise('rpc.systemManager.getSettings'),
-            Rpc.asyncPromise('rpc.skinManager.getSkinsList'),
+            //Rpc.asyncPromise('rpc.skinManager.getSkinsList'),
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/administration/MainController.js:91]: Let's just delete it, not comment it out.

File: uvm/servlets/admin/config/administration/MainController.js [hunk 3]
```diff
@@ -98,8 +98,8 @@ Ext.define('Ung.config.administration.MainController', {
             vm.set({
                 adminSettings: result[0],
                 systemSettings: result[1],
-                skinsList: result[2],
-                skinSettings: result[3]
+                //skinsList: result[2],
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/administration/MainController.js:101]: Let's just delete it, not comment it out.

File: uvm/servlets/admin/config/administration/MainModel.js [hunk 4]
```diff
@@ -9,18 +9,18 @@ Ext.define('Ung.config.administration.MainModel', {
 
         adminSettings: null,
         systemSettings: null,
-        skinSettings: null,
+        //skinSettings: null,
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/administration/MainModel.js:12]: Let's just delete it, not comment it out.

File: uvm/servlets/admin/config/administration/MainModel.js [hunk 5]
```diff
@@ -9,18 +9,18 @@ Ext.define('Ung.config.administration.MainModel', {
...
 
         serverCertificates: null,
         rootCertificateInformation: null,
         serverCertificateVerification: null,
-        skinsList: null,
+        //skinsList: null,
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/config/administration/MainModel.js:17]: Let's just delete it, not comment it out.

File: uvm/servlets/admin/config/administration/Main.js (hunk 1 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/administration/Main.js]: removed

File: uvm/servlets/admin/config/administration/MainController.js (hunk 2 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/administration/MainController.js]: removed

File: uvm/servlets/admin/config/administration/MainController.js (hunk 3 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/administration/MainController.js]: removed

File: uvm/servlets/admin/config/administration/MainModel.js (hunk 5 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/administration/MainModel.js]: removed

**Review by cblaise-untangle — APPROVED** [review]

File: uvm/servlets/admin/config/administration/MainModel.js (hunk 4 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/config/administration/MainModel.js]: Done

**Pull request merged**